aviary/examples/multi_mission/run_multimission_example_large_single_aisle.py

```diff
@@ -0,0 +1,328 @@
+"""
+authors: Jatin Soni, Eliot Aretskin
+Multi Mission Optimization Example using Aviary
+
+In this example, Two seperate aviary problems are instantiated using the typical aviaryProblem calls like load_inputs,
+check_and_preprocess_payload, etc.. Once those problems are setup and all of their phases are linked together,
+we copy those problems as group into a super_problem. We then promote GROSS_MASS, RANGE, and wing SWEEP from each
+of those sub-groups (group1 and group2) up to the super_probem so the optimizer can control them. The fuel_burn results
+from each of the group1 and group2 dymos missions are summed and weighted to create the objective function the 
+optimizer sees.
+
+For the deadhead mission: 
+aircraft:crew_and_payload:num_passengers,0,unitless
+aircraft:crew_and_payload:num_tourist_class,0,unitless
+aircraft:crew_and_payload:num_first_class,0,unitless
+
+"""
+import copy as copy
+from aviary.examples.example_phase_info import phase_info
+from aviary.variable_info.variables import Mission, Aircraft, Settings
+from aviary.variable_info.enums import ProblemType
+import aviary.api as av
+import openmdao.api as om
+import matplotlib.pyplot as plt
+from os.path import join
+import numpy as np
+import dymos as dm
+import warnings
+import sys
+from aviary.subsystems.mass.flops_based.furnishings import TransportFurnishingsGroupMass
+from aviary.api import SubsystemBuilderBase
+from aviary.validation_cases.validation_tests import get_flops_inputs
+
+# fly the same mission twice with two different passenger loads
+phase_info_primary = copy.deepcopy(phase_info)
+phase_info_deadhead = copy.deepcopy(phase_info)
+# get large single aisle values
+aviary_inputs_primary = get_flops_inputs('LargeSingleAisle2FLOPS')
+aviary_inputs_primary.set_val(Mission.Design.GROSS_MASS, val=100000, units='lbm')
+aviary_inputs_primary.set_val(Settings.VERBOSITY, val=1)
+
+aviary_inputs_deadhead = copy.deepcopy(aviary_inputs_primary)
+aviary_inputs_deadhead.set_val('aircraft:crew_and_payload:num_passengers', 1, 'unitless')
+aviary_inputs_deadhead.set_val(
+    'aircraft:crew_and_payload:num_tourist_class', 1, 'unitless')
+aviary_inputs_deadhead.set_val(
+    'aircraft:crew_and_payload:num_business_class', 0, 'unitless')
+aviary_inputs_deadhead.set_val(
+    'aircraft:crew_and_payload:num_first_class', 0, 'unitless')
+
+Optimizer = 'SLSQP'  # SLSQP or SNOPT
+
+
+class MultiMissionProblem(om.Problem):
```

Comment:
class MultiMissionProblem could live in aviary/interface eventually.

Reply:
I agree with you, but it would probably need more polish.